components/Card.js

```diff
@@ -2,22 +2,29 @@ import styled from "styled-components";
 import { useState } from "react";
 import { Button } from "./Button";
 
-export default function Card({ voc }) {
+export default function Card({ voc, handleHit, handleView }) {
   const [showTranslation, setShowTranslation] = useState(false);
 
   return (
     <StyledSection>
       <p style={{ whiteSpace: "pre-wrap" }}>
-        {voc.word}
+        {voc.word} {voc.views} {voc.hits}
         {showTranslation && `\n = \n ${voc.translation}`}
       </p>
       {!showTranslation && (
         <Button
-          onClick={() =>
+          onClick={() => {
+            handleHit(voc.id);
+            handleView(voc.id);
```

Comment:
I wonder how I will solve this, that the views are already increased during rendering. But this is only my second UserStory, so I can still think about it. 😀